server/src/main/java/com/google/android/attestation/RootOfTrust.java

```diff
@@ -51,7 +51,10 @@ public abstract class RootOfTrust {
   public abstract Optional<ByteString> verifiedBootHash();
 
   public static Builder builder() {
-    return new AutoValue_RootOfTrust.Builder();
+    return new AutoValue_RootOfTrust.Builder()
+        .setVerifiedBootKey(ByteString.EMPTY)
+        .setDeviceLocked(false)
+        .setVerifiedBootHash(ByteString.EMPTY);
```

Comment:
Why not have a default for verifiedBootState as well?

Reply:
```
public enum VerifiedBootState {
   VERIFIED,
   SELF_SIGNED,
   UNVERIFIED,
   FAILED
}
```

Is there a reasonable default in the VerifiedBootState enum?

Reply:
I vote UNVERIFIED.

Reply:
If the `deviceLocked` default is false, I would actually opt to make the `verifiedBootState` default `FAILED`. Since the default shouldn't go unchanged, I would vote for making it a state that should be impossible to reach

Reply:
I'm getting the feeling a default value for `deviceLocked` might not be a good idea and it might be better for the caller to be explicit.